### DISCUSSION TITLE
Use delayed resume overlay in editor test play

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#28438] [Ideas]
The editor's test play starts in the middle of the test play, but there is no previous cursor position.
I don't know how many mappers feel about this, but as a mapping novice, I don't feel comfortable making a mistake at the beginning of a test play either.
I think the experience would be much better if we could use the wonderful delay resumption overlay that was recently added.
Also, this should be able to be turned off from the settings.

**Comment by peppy** [comment]
Just to confirm, you mean add a countdown when entering test mode?

**Reply by shiumano** [reply]
That is what I mean.
It's a little hard to start test play after you've unreasonably lost all my health and the edges of the screen are red.
During the 3,2,1 countdown, move the cursor to the appropriate position, or in the case of the scrolling rule set, simply prepare your mind.

**Reply by peppy** [reply]
Yeah, I think this sounds fine to implement.

**Comment by shiumano** [comment]
https://github.com/ppy/osu/issues/24607
It was already there. Sorry.

**Reply by ziv-vy** [reply]
I think that was before they added the taiko/mania/catch countdown.
